tools/import-strings.sh

```diff
@@ -33,14 +33,27 @@ fi
 echo "[*] Cloning mozilla-l10n/focusios-l10n"
 git clone https://github.com/mozilla-l10n/focusios-l10n.git
 
+echo "[*] Cloning mozilla-mobile/LocalizationTools"
+[ -d tools/Localizations ] && rm -rf tools/Localizations
+git clone https://github.com/mozilla-mobile/LocalizationTools.git tools/Localizations
+
 echo "\n\n[*] Building tools/Localizations"
 (cd tools/Localizations && swift build)
 
+echo "[*] Replacing firefox with focus in swift task files"
+sed -i '' 's/firefox-ios.xliff/focus-ios.xliff/g' tools/Localizations/Sources/LocalizationTools/tasks/*.swift
+
+echo "[*] Removing es-ES locale mapping from swift import task"
+sed -i '' '/es-ES/d' tools/Localizations/Sources/LocalizationTools/tasks/ImportTask.swift
```

Comment:
Uh, good catch. One more weird thing that we should remove from that repository (we can "create" es from es-ES in the actual l10n repo).